bagit.py

```diff
@@ -1297,6 +1312,10 @@ def _make_parser():
                         help=_('Modify --validate behaviour to only test whether the bag directory'
                                ' has the number of files and total size specified in Payload-Oxum'
                                ' without performing checksum validation to detect corruption.'))
+    parser.add_argument('--completeness', action='store_true',
```

Comment:
Basically my thought is that most people are not typing these command-lines so often that one extra word is a huge burden but I'm open to being wrong about that.